telegram/ext/_contexttypes.py

```diff
@@ -33,6 +33,9 @@ class ContextTypes(Generic[CCT, UD, CD, BD]):
     Convenience class to gather customizable types of the :class:`telegram.ext.CallbackContext`
     interface.
 
+    .. seealso:: `contexttypesbot.py <https://github.com/python-telegram-bot/
+        python-telegram-bot/blob/master/examples/contexttypesbot.py>`_
+
```

Comment:
```suggestion
    .. seealso:: `contexttypesbot.py <examples3contexttypesbot.html>`_

```

the other links use this - I take it this was tested?

Reply:
yes